<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>690</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <stylestrategy>PreferAntialias</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Convert Sample Rate</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/images/sine_sweep-32x32-buttonized.png</normalon>
   </iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="FancyLineEdit" name="InfileEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>611</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Input File specification:
Wildcards (*) may be used in the filename or file extension
to process multiple files.</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="FancyLineEdit" name="OutfileEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>611</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Output File specification:&lt;/p&gt;&lt;p&gt;Wildcards (*) may be used in the filename or file extension, provided they are also used in the input file specification.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="browseInfileButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>30</y>
      <width>78</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Select an input file for processing.</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Browse ...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="browseOutfileButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>78</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Select location and name of output file.</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Browse ...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SamplerateCombo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>86</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set the output sample-rate.
(Note: you can type in a non-standard rate, if desired)</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>8000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22050</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>44100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>48000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>176400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192000</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="BitDepthCombo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>No Change</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32f</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>64f</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>alaw</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>g721-32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>gsm610</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ima-adpcm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ms-adpcm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>u8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ulaw</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="InfileLabel">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutfileLabel">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>60</y>
      <width>571</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Output File:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="NormalizeCheckBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>108</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set the peak level of the output file
(1.0 is maximum)</string>
    </property>
    <property name="text">
     <string>Normalize:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NormalizeAmountEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>130</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enter an amount between 0 and 1.0</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0.999</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DoublePrecisionCheckBox">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>130</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Use 64-bit Floating-Point arithmetic during conversion
(instead of 32-bit single-precision)</string>
    </property>
    <property name="text">
     <string>Double Precision</string>
    </property>
   </widget>
   <widget class="flashingPushbutton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>79</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Begin conversion using files and settings as currently displayed.</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ConverterOutputText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>181</y>
      <width>701</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Notifications from the converter will appear in here.</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="SampleRateLabel">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>108</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>Sample rate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConverterOutputLabel">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Converter output:</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>691</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Status: Ready</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="BitDepthCheckBox">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>108</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set the output file sub-format.
(Available sub-formats will depend on file type)</string>
    </property>
    <property name="text">
     <string>Bit Format:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DitherAmountEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Specify the number of bits of dither to add
(1.0 is default)</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DitherCheckBox">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>108</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Generate dither.
Only recommended when converting to a lower bit depth
(eg: 24bit-&gt;16bit or 16bit-&gt;8bit)</string>
    </property>
    <property name="text">
     <string>Dither:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AutoBlankCheckBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>153</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string>Deactivate dither during periods of digital silence (zeros)</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Autoblank</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="minPhase_radioBtn">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>153</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Minimum Phase Filter:
No delay, but will introduce phase distortion.
Post- ringing only.</string>
    </property>
    <property name="text">
     <string>Minimum Phase</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LinearPhase_radioBtn">
    <property name="geometry">
     <rect>
      <x>428</x>
      <y>130</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Linear Phase Filter:&lt;/p&gt;&lt;p&gt;No phase distortion, but some latency (delay).&lt;/p&gt;&lt;p&gt;Pre- and Post- ringing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Linear Phase</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ConverterTypeLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>108</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selects the type of impulse response used in the Lowpass Filter
Each type has its advantages and disadvanatges.
Linear Phase is the default. (If you are unsure, use Linear Phase)</string>
    </property>
    <property name="text">
     <string>Converter type:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="styleSheet">
     <string notr="true">QMenu {
color: #fff;
border: 1px solid #1b2018;
border-radius: 9px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #46503f, stop: 1 #2d3328);

min-width: 66px;
}

QMenu:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #535e4a, stop: 1 #3a4234);
}

QMenu:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #70e01a, stop: 1#3b770e);
}</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuCompression_Levels">
     <property name="title">
      <string>Compression Levels</string>
     </property>
     <addaction name="actionFlac"/>
     <addaction name="actionOgg_Vorbis"/>
    </widget>
    <addaction name="actionConverter_Location"/>
    <addaction name="actionOutput_File_Options"/>
    <addaction name="menuCompression_Levels"/>
    <addaction name="actionEnable_Tooltips"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="styleSheet">
     <string notr="true">QMenu {
color: #fff;
border: 1px solid #1b2018;
border-radius: 9px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #46503f, stop: 1 #2d3328);

min-width: 66px;
}

QMenu:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #535e4a, stop: 1 #3a4234);
}

QMenu:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #70e01a, stop: 1#3b770e);
}</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConverter_Location">
   <property name="text">
    <string>Converter Location ...</string>
   </property>
  </action>
  <action name="actionOutput_File_Options">
   <property name="text">
    <string>Output File Options ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt ...</string>
   </property>
  </action>
  <action name="actionEnable_Tooltips">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Tooltips</string>
   </property>
  </action>
  <action name="actionFlac">
   <property name="text">
    <string>flac ..</string>
   </property>
  </action>
  <action name="actionOgg_Vorbis">
   <property name="text">
    <string>Ogg Vorbis ...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>flashingPushbutton</class>
   <extends>QPushButton</extends>
   <header>flashingpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
